src/GitReportCL/Validation.cs

```diff
@@ -0,0 +1,81 @@
+﻿
+
+namespace GitReportCL
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    class Validation
```

Comment:
Consider how to separate validation from Console input.

Validation code **should not** have `Console.` methods in it.